game_event/models.py

```diff
@@ -17,3 +20,27 @@ class GameEvent(models.Model):
         validators=[MinLengthValidator(4)],
         choices=BallGame.choices
     )
+
+    @staticmethod
+    def create(time, level_of_game, min_number_of_players, max_number_of_players, court, ball_game):
+        GameEvent.validate_game_event(court, ball_game, min_number_of_players, time)
+        game_event = GameEvent(time=time,
+                               level_of_game=level_of_game,
+                               min_number_of_players=min_number_of_players,
+                               max_number_of_players=max_number_of_players,
+                               court=court,
+                               ball_game=ball_game)
+        game_event.save()
+        return game_event
+
+    @staticmethod
+    def validate_game_event(court, ball_game, min_number_of_players, time):
+        errors = []
+        if not CourtBallGame.is_ball_game_playable(court, ball_game):
+            errors.append("The selected court doesn't support the selected ball game.")
+        if min_number_of_players < 1:
+            errors.append("The minimum number of players must be at least 1.")
+        if time < timezone.now():
+            errors.append("The selected time must be in the future.")
```

Comment:
Maybe consider to add upper bound about the time of the event.
I think maybe to add event just in the current month. (Cause I think to start some event in the far future, like more than half a year isn't so good, it could mess up the web). 
Would like to hear your opinion about it.

Reply:
i will change it to a month interval

Reply:
> i will change it to a month interval

I'm not sure you understand me correct so just for ensuring.
I mean to add another validate about the future (not the past), like something like this:
`if time.month > timezone.now().month+1: error`
this validate that the event will be in the current month or month after and not be held about 6 month after...